content/onboarding/learners/policies-submission/_index.md

```diff
@@ -12,4 +12,9 @@ tags:
 - docx
 - close_on_peer_reviews
 title: Policy Questionaire
----
\ No newline at end of file
+---
+
+#**Umuzi Learner Agreement**
+
+[Click here](https://docs.google.com/document/d/19ZkzPCYhovIToiQ1G6L-zODFR8qSYfF6JaQvxcC0Rw0/edit) to access the Umuzi Learner Agreement before proceeding to the next card.
```

Comment:
Not sure if this is the best place for this, I was thinking that this could be a submission that checks that they read the polices and understand some key points

Reply:
Removed. I'll send it to them in an email this week.